libs/features/mas/src/variants/ccd-suggested.js

```diff
@@ -163,7 +163,11 @@ export class CCDSuggested extends VariantLayout {
         :host([variant='ccd-suggested']) ::slotted([slot='price']) {
             font-size: var(--consonant-merch-card-body-xs-font-size);
             line-height: var(--consonant-merch-card-body-xs-line-height);
-            min-width: fit-content;
+        }
+
+        :host([variant='ccd-suggested']) ::slotted([slot='price']) em {
```

Comment:
I'm not sure this style will apply.  AFAIK shadow dom styles do not apply external DOM elements except the slotted ones.

Reply:
Correct, I have removed it.